WARNING: This builds and the tests run, but I'm getting a class not found exception when using the plugin in my project. 

build.gradle:
```

buildscript {
    repositories {
        flatDir name: 'lib', dirs: "./lib"
    }
    dependencies {
        classpath 'de.schablinski:activejdbc-gradle-plugin:1.3'
    }
}

apply plugin: 'groovy'	
apply plugin: 'java'	
apply plugin: 'war'
apply plugin: 'de.schablinski.activejdbc-gradle-plugin'

activejdbc.toolVersion = 2.3

repositories {
    mavenCentral()
    jcenter()
    maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
    maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases/' }
}

dependencies {
    compile group: 'org.codehaus.groovy', name: 'groovy-all', version: '2.5.8'
    compile group: 'com.sparkjava', name: 'spark-core', version: '2.9.1'
    compile group: 'com.sparkjava', name: 'spark-template-thymeleaf', version: '2.7.1'
    compile group: 'org.thymeleaf', name: 'thymeleaf', version: '3.0.11.RELEASE'
    compile group: 'org.apache.logging.log4j', name: 'log4j-slf4j-impl', version: '2.12.1'

    compile group: 'org.pac4j', name: 'pac4j-saml', version: '3.7.0'
    compile group: 'org.pac4j', name: 'spark-pac4j', version: '3.0.0'

    compile group: 'org.javalite', name: 'activejdbc', version: '3.0-SNAPSHOT'
    implementation group: 'org.postgresql', name: 'postgresql', version: '42.2.6.jre7'
    implementation 'org.javalite:activejdbc-gradle-plugin:2.2'
}
```
```
> ./gradlew --stacktrace compileGroovy
...
Caused by: java.lang.NoClassDefFoundError: org/javalite/instrumentation/Main
        at de.schablinski.gradle.activejdbc.ActiveJDBCInstrumentation$_runInstrumentation_closure1.doCall(ActiveJDBCInstrumentation.groovy:83)
```

--- a/build.gradle
+++ b/build.gradle
@@ -10,7 +10,7 @@ plugins {
 group = 'de.schablinski.gradle.activejdbc'
 description = """Gradle plugin for instrumenting ActiveJDBC model classes."""
 
-def activeJdbcVersion = '2.3'
+def activeJdbcVersion = '2.2'
 
 repositories {
 

--- a/src/main/groovy/de/schablinski/gradle/activejdbc/ActiveJDBCInstrumentation.groovy
+++ b/src/main/groovy/de/schablinski/gradle/activejdbc/ActiveJDBCInstrumentation.groovy
@@ -20,7 +20,7 @@ import org.gradle.api.file.FileCollection
 import org.gradle.api.tasks.Classpath
 import org.gradle.api.tasks.TaskAction
 import org.gradle.process.JavaExecSpec
-import org.javalite.instrumentation.Main
+//import org.javalite.instrumentation.Main
 
 /**
  * Gradle task for performing ActiveJDBC instrumentation on a set of compiled {@code .class} files.
@@ -78,9 +78,13 @@ class ActiveJDBCInstrumentation extends DefaultTask {
     private void runInstrumentation(FileCollection instrumentationClasspath, String outputDirpath) {
         project.javaexec { JavaExecSpec jes ->
             logger.info "Running ActiveJDBC instrumentation instance with environment: ${jes.environment}"
-
+            logger.info "Implementation classpath:"
+            instrumentationClasspath.forEach() { f ->
+                logger.info(f.getAbsolutePath())
+            }
             jes.classpath = instrumentationClasspath
-            jes.main = Main.canonicalName
+//            jes.main = Main.canonicalName
+            jes.main = 'org.javalite.instrumentation.Main'
             jes.systemProperties = ['outputDirectory': outputDirpath]
         }
     }

